tests/upgrades/test_repository.py

```diff
@@ -203,3 +203,82 @@ def test_post_scenario_custom_repo_check(self, target_sat, pre_upgrade_data):
         )[0]
         result = rhel_client.execute(f'yum install -y {FAKE_4_CUSTOM_PACKAGE_NAME}')
         assert result.status == 0
+
+    @pytest.mark.pre_upgrade
+    def test_pre_scenario_custom_repo_check_sca_toggle(
+        self, target_sat, sat_upgrade_chost, save_test_data, default_location
+    ):
+        """This is a pre-upgrade scenario test to verify that repositories in a non-sca org
+        set to "Enabled" should be overridden to "Enabled(Override)" when upgrading to 6.14.
```

Comment:
Would you like to limit this only for 6.14 ?

Reply:
for now yes